draft-ietf-sidrops-publication-server-bcp-01.md

```diff
@@ -129,6 +129,12 @@ Using a unique hostname will allow the operator to use dedicated infrastructure
 and/or a Content Delivery Network for its RRDP content without interfering with
 the other functions.
 
+## Same Origin URIs
+
+Publication Servers MUST follow the normative requirements documented in section
+3.1 of [@!I-D.ietf-sidrops-rrdp-same-origin]. In short this means that URIs must
+all use the same host and redirects are not allowed.
```

Comment:
Given that the same-origin draft updates RFC 8182, and that affects all implementors of RRDP, what is the practical effect of the 'MUST' here?

Reply:
Yeah, I guess you are right and formally there is no effect.

My intention was to mention this here explicitly because implementation may not be aware of the draft.

What do you think? Should we leave this out, or change:

```
Publication Servers MUST follow the normative requirements documented
```

to, for example:
```
Publication Servers need to take note of the normative updates to [@!RFC8182] in section...
```

Better text welcome... Or should we leave it out?

Reply:
I think having a pointer is a good idea, so the suggested change ("... need to take note ...") sounds good.